bzt/modules/jmeter.py

```diff
@@ -1719,10 +1720,16 @@ def compile_http_request(self, request):
         else:
             body = request.body
 
+        use_random_host_ip = request.priority_option('use-random-source-ip', default=False)
```

Comment:
can we shorten it to "random-source-ip" ?
And also we need to document this option and its behavior.